feature/system/gnmi/cliorigin/ate_tests/mixed_oc_cli_origin_support_test/mixed_oc_cli_origin_support_test.go

```diff
@@ -119,7 +127,7 @@ func TestCLIBeforeOpenConfig(t *testing.T) {
 
 	// Validate that DUT port-1 description is `"from oc"`
 	got := gnmi.Get(t, dut, gnmi.OC().Interface(dp.Name()).Description().State())
-	want := "from oc"
+	want := "from cli"
```

Comment:
Test (requirement) readme specifically asks for  "from oc", this can not be changed.